util/MultiplayerCommon.h

```diff
@@ -169,30 +169,36 @@ struct FO_COMMON_API PlayerSaveGameData : public PlayerSaveHeaderData {
         PlayerSaveHeaderData(),
         m_orders(),
         m_ui_data(),
-        m_save_state_string()
+        m_save_state_string(),
+        m_ready(false)
     {}
 
     PlayerSaveGameData(const std::string& name, int empire_id,
                        const std::shared_ptr<OrderSet>& orders,
                        const std::shared_ptr<SaveGameUIData>& ui_data,
                        const std::string& save_state_string,
-                       Networking::ClientType client_type) :
+                       Networking::ClientType client_type,
+                       bool ready) :
         PlayerSaveHeaderData(name, empire_id, client_type),
         m_orders(orders),
         m_ui_data(ui_data),
-        m_save_state_string(save_state_string)
+        m_save_state_string(save_state_string),
+        m_ready(ready)
     {}
 
     std::shared_ptr<OrderSet>       m_orders;
     std::shared_ptr<SaveGameUIData> m_ui_data;
     std::string                     m_save_state_string;
+    bool                            m_ready;
```

Comment:
specify a default `= false;` value here, for clarity.

Reply:
Should it be changed? I don't see in other classes other than initializer list initialization.

Reply:
They are both used in various places. I'm currently preferring to move towards default-at-declare rather than in-initializer-list, but it's not important for a single line if the surrounding code uses the other method.